### ISSUE TITLE
[Others]Final exam

### ISSUE THREAD

**Issue by sjy268** [goldshakil/ElectricalProgrammingCourse2020#300]
According to another issue, we have to add percentage and grade columns.
By program code, is it okay to add student info to the original file?
That is, after executing code, is it okay that the original file is changed?

**Comment by goldshakil** [comment]
no, if you change the original file you will get a lower score.

you have to store the percentage and grade columns within your code.

**Comment by sjy268** [comment]
Do you mean that the file itself should never change under any circumstances?
So all we have to do is to refer to the file right?

**Comment by goldshakil** [comment]
yes, just read the file and never write to it.

**Comment by sjy268** [comment]
.
